docs/source/dataframe.rst

```diff
@@ -174,8 +174,14 @@ You should probably stick to just using pandas if ...
 Examples
 --------
 
-Visit https://examples.dask.org/dataframe.html to see and run examples using
-Dask DataFrame.
+Dask DataFrame is used across a wide variety of applications — anywhere where working
+with large tabular dataset. Here are a few large-scale examples:
+
+- `Parquet ETL with Dask DataFrame <https://docs.coiled.io/user_guide/usage/dask/uber-lyft.html?utm_source=dask-docs&utm_medium=dataframe>`_
+- `XGBoost model training with Dask DataFrame <https://docs.coiled.io/user_guide/usage/dask/xgboost.html?utm_source=dask-docs&utm_medium=dataframe>`_
+- `Visualize 1,000,000,000 points <https://docs.coiled.io/user_guide/usage/dask/datashader.html?utm_source=dask-docs&utm_medium=dataframe>`_
+
+You can also visit https://examples.dask.org/dataframe.html for a collection of additional examples.
```

Comment:
```suggestion
These examples scale Dask with `Coiled <https://coiled.io/?utm_source=dask-docs&utm_medium=dataframe>`_, which is one of many options for :doc:`deploying Dask <deploy>`. You can also visit https://examples.dask.org/dataframe.html for a collection of additional examples.
```

Thoughts on adding in a sentence like this?

Reply:
Thanks @scharlottej13. At first I thought this was an awkward place to mention deployment. But after thinking about it a bit, I think you're right that this is a natural place to mention it since we're talking about large-scale examples.

I chose a slightly different working to emphasize that there are many other deployment options beyond Coiled -- let me know that you think.